json/fa-token-mapping.json

```diff
@@ -1,5 +1,25 @@
 {
   "dnd5e.monsters": {
+    "irz834sqAxRihtSG": {
+      "actor":"systems/dnd5e/tokens/dragon/CopperDragonAdult.webp",
+      "token": {
+        "texture": {
+          "src": "systems/dnd5e/tokens/dragon/CopperDragonAdult.webp",
+          "scaleX": 2.3,
```

Comment:
One thing I forgot to mention that was important is that we get these scales from the filenames of the token. The original filename should have the scale number in there. You may have discovered that already, but I just wanted to double-check.

Reply:
You are correct, and the file indicated scaling will be applied next commit 
![image](https://github.com/foundryvtt/dnd5e/assets/14878515/d31a524e-2ce8-466b-a5e1-a880653111e5)